docs/reference/cluster/update-settings.asciidoc

```diff
@@ -45,6 +45,11 @@ Cluster wide settings can be returned using:
 curl -XGET localhost:9200/_cluster/settings
 --------------------------------------------------
 
+If the same setting is made both in the elasticsearch.yml configuration
+file and as a persistent cluster setting, the persistent cluster file
+setting will take presedence.  For this reason it is preferrable to 
```

Comment:
ahhh.... thank you!  Fixed.

On Thu, Nov 5, 2015 at 3:31 PM Luca Cavanna <notifications@github.com>
wrote:

> In docs/reference/cluster/update-settings.asciidoc
> <https://github.com/elastic/elasticsearch/pull/14559#discussion_r44016813>
> :
>
> > @@ -45,6 +45,11 @@ Cluster wide settings can be returned using:
> >  curl -XGET localhost:9200/_cluster/settings
> >  --------------------------------------------------
> >
> > +If the same setting is made both in the elasticsearch.yml configuration
> > +file and as a persistent cluster setting, the persistent cluster file
> > +setting will take presedence.  For this reason it is preferrable to
>
> yea my comment was to say that you wrote presedence instead of precedence
> :D
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/elasticsearch/pull/14559/files#r44016813>.
>